tools/PI/DevHome.PI/Helpers/ServicingHelper.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.UI.Xaml;
+using Windows.Win32;
+using Windows.Win32.Foundation;
+using WinUIEx.Messaging;
+
+namespace DevHome.PI.Helpers;
+
+// Note: instead of making this class disposable, we're disposing the WindowMessageMonitor in
+// UnregisterHotKey, and MainWindow calls this in its Closing event handler.
+#pragma warning disable CA1001 // Types that own disposable fields should be disposable
+public class ServicingHelper
+#pragma warning restore CA1001 // Types that own disposable fields should be disposable
+{
+    private const string NoWindowHandleException = "Cannot get window handle: are you doing this too early?";

```

Comment:
NIT: Move the const into `ServicingHelper` because it isn't used by anything else.